### ISSUE TITLE
loss.py is missing

### ISSUE THREAD

**Issue by jaehong31** [CharlesGong12/RECE#1]
Hi authors,

Congrats on publishing amazing work. When I try to run the code,

https://github.com/CharlesGong12/RECE/blob/f9a40445f3d0f5c7b2aa8c5c784f55290525aef9/utils/__init__.py#L1C1-L2C1

This loss function is not defined or missing in the repo. 

Could you check this?

Thanks in advance

**Comment by CharlesGong12** [comment]
Hi, thanks for your attention to our work.

Code about the loss function is not used and it was for gradient-based adversarial training, like PGD, in my early exploring stage. I deleted the related code and updated the git repository. The code can be directly run now. 

Any further questions, plz don't hesitate and contact me.
